modules/KalturaSupport/UiConfResult.php

```diff
@@ -209,7 +209,7 @@ public function parseJSON( $uiConf ) {
 
 		// Add Core plugins
 		$basePlugins = array(
-			'statistics' => array(),
+//			'statistics' => array(),

```

Comment:
@OrenMe, it would be helpful if the comment marks will be right before the name of the plugin.

Reply:
done